grama/eval_defaults.py

```diff
@@ -240,7 +360,8 @@ def eval_conservative(model, quantiles=None, df_det=None, append=True, skip=Fals
             of values for each random variable, or None for default 0.01.
             values in [0, 0.5]
         df_det (DataFrame): Deterministic levels for evaluation; use "nom"
-            for nominal deterministic levels.
+            for nominal deterministic levels. (None) accepted if 
```

Comment:
Probably more lucid to write `If provided model has no deterministic variables (model.n_var_det == 0), then df_det may equal None.` Note that `model.n_var_det` is an implementation detail that is not so user-facing.